src/signals/incident/definitions/wizard-step-2-vulaan/wegen-verkeer-straatmeubilair.ts

```diff
@@ -3,7 +3,16 @@
 import { FIELD_TYPE_MAP } from 'signals/incident/containers/IncidentContainer/constants'
 
 import appConfiguration from 'shared/services/configuration/configuration'
+import type { IconOptions } from 'leaflet'
 import type ConfigurationType from '../../../../../app.amsterdam.json'
+import * as verlichtingIcons from './verlichting-icons'
+
+export const ICON_SIZE = 40
+
+const options: Partial<IconOptions> = {
```

Comment:
Why partial? That makes every prop optional. If both `className` and `iconSize` aren't optional, can't `Pick<IconOptions, 'className | iconSize'>` be used?

Reply:
sure